lib/model/rwfolder.go

```diff
@@ -1532,7 +1540,7 @@ func (f *sendReceiveFolder) dbUpdaterRoutine() {
 			}
 			lastFile = file
 			if err := syncFn(file); err != nil {
```

Comment:
The above changes allow ``file`` to traverse a Symlink. You have to check with ``isDir``, before calling ``syncFn``. It prevents fsyncing files outside of the folder and the fsync errors disappear.